lib/resque/job.rb

```diff
@@ -227,6 +230,9 @@ def args
       @payload['args']
     end
 
+    # Returns an array of args which is the args after all hooks have run
+    attr_reader :final_args
+    
```

Comment:
If this is just for tests, let's add it in the test case itself rather than exposing publicly.